DCRack/dc_rack_server.py

```diff
@@ -273,7 +272,6 @@ def changeLEDs(self, data):
             data = 4*data[0] + 2*data[1] + 1*data[2]
         else:
             data &= 0x7
-        self.write([1L])
```

Comment:
It would be nice if someone could comment on why this was here in the first place. @zchen088 @thchwhite?